app/controllers/reviews_controller.rb

```diff
@@ -18,6 +18,40 @@ def create
     end
   end
 
+  def edit
+    @beach = Beach.find(params[:beach_id])
+    @review = Review.find(params[:id])
+  end
+
+  def update
+    @beach = Beach.find(params[:beach_id])
+    @user = current_user
+    @review = Review.find(params[:id])
+    @review.user = @user
+    if @review.update(review_params)
+      flash[:notice] = "Review updated successfully."
+      redirect_to beach_path(@beach)
+    else
+      flash.now[:notice] = "Review not updated successfully. " +
+                            @review.errors.full_messages.join(". ")
```

Comment:
Use 2 (not 22) spaces for indenting an expression in an assignment spanning multiple lines.